### PR TITLE
Test judge pipeline

### LINKED ISSUE #121: Add more sample data to tests in judges pipeline
## Task Description

- Some tests could use a little more test data - particularly the parametrized tests.

## User Stories

- As a developer, I need to make sure my tests cover a large ground, so that I know that I'm dealing with edge cases

## Relevant Files [If Available] 

- `test_pipeline.py`

### PR DESCRIPTION
**Pull request by luciegamby-sigmalabs**
Added more test data, also included new tests for fuzzy_match_circuit.

Closes #121

### CODE REVIEW

File: judges/test_pipeline.py [hunk 1]
```diff
@@ -34,7 +36,15 @@ def test_convert_date_returns_datetime():
...
-                                                                          ("Their Honour Judge Foobar", "Honour Judge", "Foobar", "X")])
+                                                                          ("Their Honour Judge Foobar",
+                                                                           "Honour Judge", "Foobar", "X"),
+                                                                          ("Mr Justice Fizz",
+                                                                           "Justice", "Fizz", "M"),
+                                                                          ("Mrs Justice Buzz",
+                                                                           "Justice", "Buzz", "F"),
+                                                                          ("Miss Justice Fizzbuzz",
+                                                                           "Justice", "Fizzbuzz", "F")
+                                                                          ])
```
Comment (MahinRahman8901) [inline judges/test_pipeline.py:47]: perfect, now the tests will be more accurate

**Review by MahinRahman8901 — APPROVED** [review]
